exchanges/binance/binance.go

```diff
@@ -911,10 +911,8 @@ func (b *Binance) SendAuthHTTPRequest(ctx context.Context, ePath exchange.URL, m
 
 // CheckLimit checks value against a variable list
 func (b *Binance) CheckLimit(limit int) error {
-	for x := range b.validLimits {
-		if b.validLimits[x] == limit {
-			return nil
-		}
+	if slices.Contains(b.validLimits, limit) {
+		return nil
 	}
 	return errors.New("incorrect limit values - valid values are 5, 10, 20, 50, 100, 500, 1000")
```

Comment:
🐒 Nitpick:
* If we're going to quote anything, it should be what the supplied value was
* I'm against prompting what valid values are in errors, but if we must, we should `strings.Join()` on `b.validLimits`
```suggestion
	return fmt.Errorf("%w `%v`, errInvalidLimit, limit)
```

Reply:
It turns out that Binance allows any limit value between 1-5000 so I've removed it entirely for it and BinanceUS https://github.com/thrasher-corp/gocryptotrader/pull/1826/commits/bbbe32bcb856851d2819794b2d652c1b6278432c